core/src/main/java/org/apache/iceberg/util/ThreadPools.java

```diff
@@ -56,6 +56,22 @@ public static ExecutorService getWorkerPool() {
     return WORKER_POOL;
   }
 
+  /**
+   * Returns delete worker pool for Positional Deletes.
+   *
+   * <p>Size of this thread-pool is controlled by System Property {@code
+   * iceberg.delete.pos.read-worker-pool}.
+   *
+   * @return {@link ExecutorService} that is delete worker pool
+   */
+  public static ExecutorService newDeleteWorkerPool() {
```

Comment:
If this is going to behave like the worker pool, then it should also be a static, shared pool. Otherwise, this will create multiple pools that have the same name.

It should either behave like the existing `getWorkerPool` or like the existing `newWorkerPool` that requires a name.